gson/src/main/java/com/google/gson/JsonArray.java

```diff
@@ -28,11 +29,14 @@
  * elements are added is preserved. This class does not support {@code null} values. If {@code null}
  * is provided as element argument to any of the methods, it is converted to a {@link JsonNull}.
  *
+ * <p>{@code JsonArray} only implements the {@link Iterable} interface but not the {@link List}
+ * interface, but a {@code List} view of it can be obtained with {@link #asList()}.
```

Comment:
I would start a new sentence here.
> ...interface. A {@code List} view...

Reply:
Thanks! I wasn't really happy with the previous wording either.